### ISSUE TITLE
Success Criterion 1.3.4 - Orientation - Level A

### ISSUE THREAD

**Issue by JJdeGroot** [w3c/matf#25]
Discussion:

<details>
<summary>MATF meeting on July 3, 2024</summary>

[Source](https://www.w3.org/2024/06/26-matf-minutes.html#t04)

jj: the examples of "essential" are outdated

<Jamie> +1 to JJ that the examples in WCAG are not essential

<quintinb> Should this be AA? I would argue A

jj: what would be the situations where an orientation is essential?

<Joe_Humbert> https://www.w3.org/TR/WCAG22/#orientation is Level AA

<Jamie> I was wrong, this is AA; @JJ shold we discuss later

quintinb: how it was defined for web? why like this?

quintinb: this should be an A for mobile

<Joe_Humbert> it was added in 2.1

<Detlev> I think the main ides was to support web on mobile devices better

AlainVagner: very common issue, could have some techniques to help developers

<Joe_Humbert> I have to drop I'll bring it up next timne

jj: issues will be created upfront

jj: will think about a new organisation
</details>

Summary:

**1. Essential:**
- JJ: The examples of what is considered "essential" in the WCAG guidelines are outdated / not relevant in context of mobile apps
- Jamie: Agreed with JJ that the examples in WCAG are not essential.

**2. Discussion:**
- Quintinb: Questioned how orientation was defined for the web and why it was defined in that way.
- Joe_Humbert: Mentioned that this criterion was added in WCAG 2.1.
- Detlev: Believed the main idea was to better support web on mobile devices.
- AlainVagner: Mentioned that this is a common issue and proposed developing techniques to help developers ensure compliance.

Joe and Jamie will bring up their thoughts in next weeks' meeting due to time.

**Comment by qbalsdon** [comment]
> Some websites and applications automatically set and restrict the screen to a particular display orientation and expect that users will respond by rotating their device to match, but this can create problems. Some users have their devices mounted in a fixed orientation (e.g. on the arm of a power wheelchair). Therefore, websites and applications need to support both orientations by not restricting the orientation. Changes in content or functionality due to the size of display are not covered by this criterion which is focused on restrictions of orientation. [Reference][0]

This alone is sufficient for me to agree that this should remain consistent.

[0]: https://www.w3.org/WAI/WCAG21/Understanding/orientation.html

**Comment by JJdeGroot** [comment]
Related discussion: https://github.com/w3c/wcag/issues/613

**Comment by JJdeGroot** [comment]
Related Reddit thread: https://www.reddit.com/r/androiddev/s/IdCrOT1Jur

**Comment by JJdeGroot** [comment]
Discussed in today’s meeting.

<details>
<summary>13 November 2024</summary>

[Source](https://www.w3.org/2024/11/13-matf-minutes.html#fe68)

~~~text
Aash did not understand the 4 orientations concepts?

Joe_Humbert No more work needed to support all 4 orientations. It is same rendering in terms of pixels

JJ it is a best practice to support all 4 orientations.

Illai should we propose this in conjunction with target size SC?

<JJ> Should we add a note to 1.3.4 about the best-practice of supporting all 4 (or more) orientations of a device?

JJ we can add 4 orientation in notes, along with the target size SC, basic and advanced.

<Illai> +1

<Aash> +1

<Karla> +1

<GleidsonRamos> +1

<Tim> +1

ACTION: Add a note to 1.3.4 about the best-practice of supporting all 4 (or more) orientations of a device
~~~
</details>

Conclusion: we are going to propose text for 1.3.4 (as is) and keep note 1 of WCAG2ICT.

**Comment by alastc** [comment]
Something I've found our clients really struggle with (or at least, I struggle to justify) is where an app is restricted in the same way as the operating system.

For example, on small screen (phone) iOS device the homescreen is locked to portrait. Whenever I've seen these devices mounted they are in the default orientation, and that makes sense because why would you adjust that from the homescreen default?

Forcing a landscape orientation is obviously going to be a problem, e.g. for video.

However, shouldn't we add / encourage an exception for when the device/OS  defaults to a particular orientation?

**Comment by JJdeGroot** [comment]
Hi @alastc , it's correct that the iOS home screen is locked to portrait. Because of this limitation, it's unlikely that someone without the dexterity to rotate their phone would mount it in landscape orientation. However, I can imagine that a caretaker would put it in landscape orientation to watch video. 

I agree that changing between apps can be problematic if the OS defaults to a particular orientation. We could add an exception for those situations, but I would add that on the app-level, not on the system-level.

What I did notice is that Apple has now restricted orientation of iOS apps to 3 orientations:`portrait`, `landscapeLeft` and `landscapeRight`. The `upsideDown` orientation can no longer be used on newer iPhones. See also: https://discussions.apple.com/thread/253644885?sortBy=rank&page=1

Upside down had one use case that I was personally using: charging my phone in the cup holder of my car, placing it upside down to avoid breaking the connector. This is also mentioned numerous times in the Apple discussion thread.

Providing users with full freedom of orientations empowers them to use their devices according to their needs.

**Comment by JJdeGroot** [comment]
Starting from Android 16, fixed orientation (e.g. portrait only) is no longer supported for larger screens, meaning that developers have to support a larger variety of orientations: https://android-developers.googleblog.com/2025/01/orientation-and-resizability-changes-in-android-16.html?m=1 (There has been no better time to push for Success Criterion 1.3.4)

**Comment by JJdeGroot** [comment]
Discussed in yesterday’s meeting.

<details>
<summary>16 July 2025</summary>

[Source](https://www.w3.org/2025/07/16-matf-minutes.html#9225)

~~~text
1.3.4 Orientation

<JJ> [w3c/matf#25](https://github.com/w3c/matf/issues/25)

JJ: Orientation was added to Draft in February then reopened

<JJ> Success Criterion 1.4.4 Orientation

<JJ> This applies directly as written, and as described in Intent from Understanding Success Criterion 1.3.4.

<JJ> Note 1

<JJ> Content that supports orientation changes must continue to meet all applicable success criteria in each supported orientation.

<JJ> Note 2

<JJ> It is considered a best practice to support all available orientations, such as as portrait, portrait (reversed), landscape, and landscape (reversed). Platform restrictions may limit available orientations for specific content types (such as video playback) or system-level functions.

JJ: There might be some platform restrictions with native components that would prevent supporting all orientations

<JJ> Poll: do we accept the definition proposal?

<RacheleD> +1

<Jamie> +1

<GleidsonRamos> +1

rachaely: I wanted to understand what our stance on reflow for mobile apps?

<JJ> 1.4.10: [w3c/matf#4](https://github.com/w3c/matf/issues/4)

<Joe_Humbert> 0 - i'm concerned that companies might take advantage of the last sentence in the second note, but I guess there is always the essential execption that they could try and use

<rachaely> +1

<Carol> +1

<JJ> Poll: Remove "Platform restrictions may limit available orientations for specific content types (such as video playback) or system-level functions."

<Jamie> +1

<Joe_Humbert> essential

<Joe_Humbert> if removed, would fundamentally change the information or functionality of the content, and information and functionality cannot be achieved in another way that would conform

Joe_Humbert: WCAG 2.2 says "unless a specific display orientation is essential" do we need the second sentence?

rachaely: Some things like a video player may only support landscape, not portrait. Should that be a standalone note on its own? But that may be covered

<rachaely> +1

<RacheleD> +1

ACTION: Remove "Platform restrictions may limit available orientations for specific content types (such as video playback) or system-level functions." from note

<Carol> +1

<JJ> (potentially as separate note)

Joe_Humbert: I see some validity to keeping it but it should probably be a separate note

<Jamie> + Jamie

<Jamie> Zakim

<JJ> Thanks for scribing RacheleD
~~~
</details>

- ACTION: Proposed definition is accepted, with the removal of "Platform restrictions may limit available orientations for specific content types (such as video playback) or system-level functions." from the note
